Form-Controls/index.html

```diff
@@ -13,15 +13,47 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="orderForm" onsubmit="return validateForm()">
+    <label for="name">Full Name:</label>
```

Comment:
Good job! The selection of the different fields is correct, e.g. radio buttons for the colour and a select one for the size. The date picker is also nice.

AC 1. says the following:
"What is the customer's name? I must collect this data, and validate it. But what is a valid name? I must decide something."

Is this name field validated as is now? What does this "onsubmit="return validateForm()"" do? Would this be only html and css to make it work this way?

Do you have any other idea how this form could be validated?
(hint: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_against_a_regular_expression)

Reply:
Here I added "required" and pattern to validate the name input